mailbox/tools/quota-recompute/pom.xml

```diff
@@ -36,15 +36,35 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-mailbox-store</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-json</artifactId>
+            <scope>test</scope>
+            <type>test-jar</type>
+        </dependency>
         <dependency>
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-data-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-task-json</artifactId>
```

Comment:
importing json into mailbox-api?

Reply:
That's not `mailbox-api` but `mailbox-tools-quota-recompute` module here?
Well to create the task `RecomputeCurrentQuotasTask` wrapping around the service `RecomputeCurrentQuotasService`, with its DTOs that need `james-server-task-json`? `RecomputeCurrentQuotasTaskDTO` and `RecomputeCurrentQuotasTaskAdditionalInformationDTO`)

Can you please then be more explicit in why you think it's wrong again, and what are your suggestions around that? Thanks.

Reply:
I checked twice before adding this comment: github displayed mailbox-api ... now it's no longer the case.

Reply:
There is some serious issues with github seems these days...